packages/bindings-test/src/coreum_testing_deps.rs

```diff
@@ -0,0 +1,188 @@
+use std::{collections::HashMap, marker::PhantomData};
+
+use coreum_wasm_sdk::core::CoreumQueries;
+use cosmwasm_std::{
+    from_json,
+    testing::{MockApi, MockQuerier, MockStorage, MOCK_CONTRACT_ADDR},
+    to_json_binary, Addr, Coin, Decimal, OwnedDeps, Querier, QuerierResult, QueryRequest,
+    SystemError, SystemResult, Uint128, WasmQuery,
+};
+use dex::factory::{
+    ConfigResponse, FeeInfoResponse,
+    QueryMsg::{Config, FeeInfo},
+};
+
+use cw20::{BalanceResponse, Cw20QueryMsg, TokenInfoResponse};
+
+pub type CoreumDeps = OwnedDeps<MockStorage, MockApi, SplitterMockQuerier, CoreumQueries>;
+
+pub fn mock_coreum_deps(contract_balance: &[Coin]) -> CoreumDeps {
+    let custom_qurier =
+        SplitterMockQuerier::new(MockQuerier::new(&[(MOCK_CONTRACT_ADDR, contract_balance)]));
+
+    CoreumDeps {
+        storage: MockStorage::default(),
+        api: MockApi::default(),
+        querier: custom_qurier,
+        custom_query_type: PhantomData,
+    }
+}
+
+pub struct SplitterMockQuerier {
```

Comment:
```suggestion
pub struct MockQuerier {
```

Reply:
this will conflict with the struct with same name in `cosmwasm_std::testing`, as per our conversation we agreed on renaming this to `WhelpMockQuerier`